packages/massa-web3/src/web3/PublicApiClient.ts

```diff
@@ -130,6 +131,30 @@ export class PublicApiClient extends BaseClient implements IPublicApiClient {
     return { ...nodeStatus, chain_id: BigInt(nodeStatus.chain_id) }
   }
 
+  /**
+   * Retrieves the minimal fees for operations to be accepted by the API provider.
+   *
+   * @returns The minimal fees as a string (in nano-MAS).
+   */
+  public async getMinimalFees(): Promise<bigint> {
+    const jsonRpcRequestMethod = JSON_RPC_REQUEST_METHOD.GET_STATUS
+    let fees: string = '0'
+    if (this.clientConfig.retryStrategyOn) {
+      let result = await trySafeExecute<INodeStatus>(this.sendJsonRPCRequest, [
+        jsonRpcRequestMethod,
+        [],
+      ])
+      fees = result.minimal_fees || '0'
+    } else {
+      let result = await this.sendJsonRPCRequest<INodeStatus>(
+        jsonRpcRequestMethod,
+        []
+      )
+      fees = result.minimal_fees || '0'
+    }
+    return BigInt(parseFloat(fees) * 10 ** MASSA_SCALING_FACTOR)
```

Comment:
what not using fromMas from web3-utils instead ?

Reply:
Because I forget that it exist. Fixed now.